src/index.html

```diff
@@ -1,13 +1,589 @@
+                </p>
+              </article>
+            </div>
+          </section>
+
+          <section class="contacts" id="contacts">
+
+            <h2 class="section-title contacts__title">Contact us</h2>
+            <div class="contacts__wrapper">
+
+              <div class="contacts__contact">
+
+                <div class="contact contacts__contact-info">
+                  <h3 class="contact__name">The Met Fifth Avenue</h3>
+                  <h4 class="contact__title">Phone</h4>
+                  <a href="tel:+1 212-535-7710" class="contact__value">+1 212-535-7710</a>
+
+
+                  <h4 class="contact__title">Address</h4>
+                  <a
+                    href="#"
+                    target="_blank"
+                    class="contact__info text-info"
+                  >
+                    1000 Fifth Avenue
+                    New York, NY 10028
+                  </a>
+
+                </div>
+
+                <div class="contact contacts__contact-info">
+                  <h3 class="contact__name">The Met Cloisters</h3>
+                  <h4 class="contact__title">Phone</h4>
+                  <a
+                    href="tel:+1 212-923-3700"
+                    class="contact__value"
+                  >
+                    +1 212-923-3700
+                  </a>
+
+
+                  <h4 class="contact__title">Address</h4>
+                  <a
+                    href="#"
+                    target="_blank"
+                    class="contact__info text-info"
+                  >
+                    99 Margaret Corbin Drive
+                    Fort Tryon Park
+                    New York, NY 10040
+                  </a>
+
+                </div>
+              </div>
+
+              <div class="contacts__form">
+                <form
+                  class="contact__form"
+                  onsubmit="this.reset(); return false;"
+                >
+                  <input
+                    type="text"
+                    class="contact__form__input"
+                    name="name" placeholder="Your name"
+                    required
+                  >
+                  <input
+                    type="email"
+                    class="contact__form__input"
+                    name="e-mail"
+                    placeholder="E-mail"
+                    required
+                  >
+                  <textarea
+                    class="contact__form__input contact__form__input--big"
+                    name="message"
+                    placeholder="Message"
+                    required
+                  >
+                  </textarea>
+                  <button class="contact__form__button">Send</button>
+                </form>
+              </div>
+            </div>
+          </section>
+        </div>
+      </div>
+
+      <!-- <div class="main-content">
```

Comment:
Remove comments. Try it write obvious code that explains all we need without comments)